src/Sia.Gateway/Requests/Incidents/GetIncident.cs

```diff
@@ -6,6 +6,7 @@
 using Sia.Domain;
 using Sia.Shared.Authentication;
 using Sia.Shared.Requests;
+using Sia.Shared.Protocol;
```

Comment:
I don't see why this change is needed… 👇🏼 the revision below does not introduce a dependency on shared protocol.

Reply:
https://stackoverflow.com/questions/283471/remove-unused-namespaces-across-a-whole-project-or-solution-at-once has an interesting solution to remove unnecessary usings across the whole solution; we may want to do this regularly or set up a travis rule or something that alerts us to unnecessary usings. They're relatively hard to detect manually and I know I don't often think about removing namespaces when I'm no longer using them (which may be its own issue)